middleware/content_type.go

```diff
@@ -37,11 +37,9 @@ func AllowContentType(contentTypes ...string) func(next http.Handler) http.Handl
 				s = s[0:i]
 			}
 
-			for _, t := range cT {
-				if t == s {
-					next.ServeHTTP(w, r)
-					return
-				}
+			if allowedContentTypes[s] {
```

Comment:
If all you're doing is checking existence _within_ the map, might as well use `map[string]struct{}` and then comma okay. We don't actually care about the value, since they are all true.

```
if _, ok := allowedContentTypes[s]; ok {
	next.ServeHTTP(w, r)
	return
}
```

Reply:
Done. Tested with a benchmark test and it is a few nanoseconds better